README.rst

```diff
@@ -69,6 +69,9 @@ your build tree. The following are the absolute minimum requirements:
 - Clang from Xcode 7.3
 - Visual Studio 2015 (MSVC++ 14.0)
 
+Recommended minimum required CMake version:
+- CMake 3.12.4 
```

Comment:
the minimum cmake version doesn't appear as a bullet list item. I think a space is needed between lines 72 and 73.